src/pages/ChangeUser/ChangeUser.js

```diff
@@ -13,30 +12,32 @@ export class ChangeUserContainer extends React.Component {
   }
 
 
-    submit = (fields) => {
-      this.setState({ loading: true });
+  submit = (fields) => {
+    this.setState({ loading: true });
```

Comment:
1) for what reason do you set this _loading_ flag? Noone can see it, because on submit you make `this.props.history.push('/');`

2) there is no need to redirect user to main page after you save user